doc/cheatsheet.md

```diff
@@ -0,0 +1,262 @@
+# djot cheatsheet
+
+## Basics
+
```

Comment:
One thing not mentioned in "Basics," but which probably should be:

- leave blank lines between paragraphs
- line breaks are interpreted like spaces

Reply:
:+1:

Reply:
So, I believe it's *one* line break is interpreted as a space, correct?

Reply:
Ok. Clarified this.